### DISCUSSION TITLE
关于三方项目声明“Built with TDesign”的一个询问

### DISCUSSION THREAD

**Discussion by ArcticFoxPro** [Tencent/tdesign#513] [Q&A] [answered]
想问一下，我的一些项目使用了 TDesign，我能否在项目的关于界面或者 Readme 界面放上类似于“Built with TDesign”这样的标记呢？

类似于 WordPress 的“自豪地采用 WordPress”、Discuz! 的“Powered by Discuz!”、Unity 的“Made with Unity”“Made with 团结引擎”。

由于 TDesign 及其 Logo 是商标，所以我想问一下关于这方面的使用是否被许可。

示例：

![mmexport1721205094732.jpg](https://github.com/user-attachments/assets/3934c078-62e9-4a91-bc0d-936b1a56ac2c)

**Comment by uyarn** [comment, marked as answer]
可以。

**Reply by ArcticFoxPro** [reply]
好的，谢谢您
